### PR TITLE
feat: add venv script and steps

### LINKED ISSUE #6: Switch to venv for reproducible environments
We should switch to `venv` rather than risking issues with the user's Ansible version. It is more reproducible and allows us to remove the ansible < 2.10 parts.

### PR DESCRIPTION
**Pull request by leopaul36**
### Linked issues

Fix #6 

### Changes

- Added `venv` setup script
- Removed Ansible < 2.11 commands

### CODE REVIEW

File: bin/setup.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+readonly AVAILABLE_FEATURES=(extras prerequisites vagrant server observability)
```
Comment (Nuttymoon) [inline bin/setup.sh]: What is this variable for? :thinking:

File: bin/setup.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+readonly PYTHON_BIN=${PYTHON_BIN:-python3}
+readonly PYTHON_VENV=${PYTHON_VENV:-venv}
```
Comment (Nuttymoon) [inline bin/setup.sh]: I'd rather use `.venv` for this folder

File: README.md [hunk 3]
```diff
@@ -46,19 +46,20 @@ How to use the [ash.avalanche](https://github.com/AshAvalanche/ansible-avalanche
...
 
-   # With Ansible >= 2.10
-   ansible-galaxy collection install git+https://github.com/AshAvalanche/ansible-avalanche-collection.git,v0.3.0
+3. Install the `ash.avalanche` collection:
 
-   # With Ansible 2.9
-   mkdir -p ansible_collections/ash
-   git clone --branch v0.3.0 https://github.com/AshAvalanche/ansible-avalanche-collection.git ansible_collections/ash/avalanche
+   ```sh
+   ansible-galaxy collection install git+https://github.com/AshAvalanche/ansible-avalanche-collection.git,v0.3.0
```
Comment (Nuttymoon) [inline README.md:62]: The latest version is 0.4.0

File: bin/setup.sh (hunk 1 above)
Comment (leopaul36) [reply bin/setup.sh]: Removed in https://github.com/AshAvalanche/ansible-avalanche-getting-started/pull/7/commits/1d0c2a4befc482d6aa305b6da345bef8d1b28e45

File: bin/setup.sh (hunk 2 above)
Comment (leopaul36) [reply bin/setup.sh]: Done in https://github.com/AshAvalanche/ansible-avalanche-getting-started/pull/7/commits/1d0c2a4befc482d6aa305b6da345bef8d1b28e45

**Review by Nuttymoon — APPROVED** [review]

**Pull request merged**